Documentation/README.md

```diff
@@ -6,16 +6,25 @@ The following document includes documentation for the build scripts. Over time,
 
 The scripts are supported on Windows, macOS and Linux. The scripts are based on PowerShell on Windows and Bash on macOS and Linux.
 
-### Build on Windows
+### Syncing
+
+This repo uses submodules so remember on fresh clones or when updating/switching branches to run
```

Comment:
The doc should probably also mention `git submodule sync [--recursive]` for when `.gitmodules` changes to point to different forks.

Reply:
I'm not sure that will be a common issue so we can keep it out for now.